### ISSUE TITLE
[jsonConfig] Make "trim()" optional / configurable on text fields

### ISSUE THREAD

**Issue by klein0r** [ioBroker/ioBroker.admin#1355]
**Is your feature request related to a problem? Please describe.**

- I have a text field which can contain leading and trailing spaces. This allows the user to define a custom string to concatenate text. Example: ``", "`` or ``" and "``.
- With Admin 5 all text fields are trimmed automatically. So the users cannot add any white spaces. They will be removed.

**Describe the solution you'd like**

I need an option in the ``jsonConfig.json`` text field definition, to avoid automatic ``trim()`` for values. Example:

```json
{
    "nextseparator": {
        "type": "text",
        "sm": 12,
        "md": 6,
        "lg": 6,
        "trim": false
    }
}
```

**Comment by GermanBluefox** [comment]
Implemented in next version 5.3.2

**Comment by klein0r** [comment]
Thanks a lot for this fast response! Don't forget to update https://github.com/ioBroker/adapter-react/blob/master/schemas/jsonConfig.json ✌️
